HTTP_200/settings.py

```diff
@@ -107,23 +107,23 @@
 
```

Comment:
@singhsterabhi I think you have committed this file by mistake. Please remove it as this isn't the part of this PR.